### PR TITLE
D54321

### PR DESCRIPTION
**Pull request by valerii-fed**
Description

### CODE REVIEW

File: code_review.proto [hunk 1]
```diff
@@ -69,49 +69,49 @@ message Diff {
...
-  enum Type {
-    // Thread on a code line
-    CODE = 0;
-    // Thread on commit (synced from other systems e.g GitHub)
-    COMMIT = 1;
-    // Thread on repo (synced from other systems e.g GitHub)
-    REPO = 2;
-    // Thread on whole diff
-    DIFF = 3;
-  }
```
Comment (valerii-fed) [inline code_review.proto]: Created by: **valerii.fedorenko.ua@ukr.net**
Time: **Sun Jan 18 22:24:14 EET 1970**
Created by: **valerii.fedorenko.ua@ukr.net**
Time: **Sun Jan 18 22:24:12 EET 1970**
Created by: **valerii.fedorenko.ua@gmail.com**
Time: **Sun Jan 18 22:24:12 EET 1970**
77

Comment (valerii-fed) [inline code_review.proto]: Created by: **valerii.fedorenko.ua@ukr.net**
Time: **Sun Jan 18 22:24:14 EET 1970**
Created by: **valerii.fedorenko.ua@gmail.com**
Time: **Sun Jan 18 22:24:14 EET 1970**
really 78

Comment (valerii-fed) [inline code_review.proto]: Created by: **valerii.fedorenko.ua@gmail.com**
Time: **Sun Jan 18 22:24:14 EET 1970**
72 red

File: code_review.proto [hunk 2]
```diff
@@ -69,49 +69,49 @@ message Diff {
...
+	  string bug = 6;
+	  Status status = 7;
+
+	  string workspace = 8;
+	  int64 created_timestamp = 9;
+	  int64 modified_timestamp = 10;
+	  // Threads on a line.
+	  repeated Thread code_thread = 13;
+	  repeated string cc = 14;
+	  // Threads on the whole diff, not on a particular file or line.
```
Comment (valerii-fed) [inline code_review.proto]: Created by: **valerii.fedorenko.ua@ukr.net**
Time: **Sun Jan 18 22:24:14 EET 1970**
Created by: **valerii.fedorenko.ua@gmail.com**
Time: **Sun Jan 18 22:24:14 EET 1970**
87

File: code_review.proto [hunk 3]
```diff
@@ -69,49 +69,49 @@ message Diff {
...
+
+	// Comment thread. Note that for whole-diff threads, some fields will be empty
+	// (e.g repo_id, commit_id).
+	message Thread {
+	  enum Type {
+	    // Thread on a code line
+	    CODE = 0;
+	    // Thread on commit (synced from other systems e.g GitHub)
+	    COMMIT = 1;
+	    // Thread on repo (synced from other systems e.g GitHub)
```
Comment (valerii-fed) [inline code_review.proto]: Created by: **valerii.fedorenko.ua@ukr.net**
Time: **Sun Jan 18 22:24:14 EET 1970**
Created by: **valerii.fedorenko.ua@gmail.com**
Time: **Sun Jan 18 22:24:14 EET 1970**
110

File: code_review.proto [hunk 4]
```diff
@@ -69,49 +69,49 @@ message Diff {
...
-    // Thread on a code line
-    CODE = 0;
-    // Thread on commit (synced from other systems e.g GitHub)
-    COMMIT = 1;
-    // Thread on repo (synced from other systems e.g GitHub)
-    REPO = 2;
-    // Thread on whole diff
-    DIFF = 3;
-  }
+	  int64 id = 1;
```
Comment (valerii-fed) [inline code_review.proto]: Created by: **valerii.fedorenko.ua@gmail.com**
Time: **Sun Jan 18 22:24:14 EET 1970**
72 green

**Comment by val-fed** [comment]
PR comment

**Pull request closed without merging**